### ISSUE TITLE
Base requirement

### ISSUE THREAD

**Issue by FlorianPydde** [microsoft/dstoolkit-devcontainers#8]
I have a comment regarding the requirements.txt in both src and notebook file. As you commented in the requirement.txt,  the packages to ensure good coding practice (like bandit, black, etc), are repeated across folder's req files. 

Why not simply use 1 main req file for all the linting/code checking packages and then some custom reqs for src and notebooks ? In that way we avoid risking having multiple package version.
Basically something like this:

_notebook
|-- requirement-notebooks.txt
src
|-- requirement-src.txt
requirement-ops.txt (or whatever name)_
